src/Symbols/include/Symbols/SymbolUtils.h

```diff
@@ -26,11 +26,15 @@ namespace orbit_symbols {
 // ErrorMessage otherwise.
 [[nodiscard]] ErrorMessageOr<void> VerifySymbolFile(const std::filesystem::path& symbol_file_path,
                                                     const std::string& module_build_id);
-// Checks if the file at symbol_file_path can be read as symbol file (elf, coff, pdb) and compares
-// the size of the file with expected_file_size. Returns void if the sizes are the same ErrorMessage
+// Checks if the file at object_file_path can be read as an object file (elf, coff) and compares the
+// build id of the file with module_build_id. Returns void if build ids are the same, ErrorMessage
```

Comment:
```suggestion
// build id of the file with build_id. Returns void if build ids are the same, ErrorMessage
```

Reply:
I actually fixed the parameter names across a couple of functions as they were inconsistent (including between declaration and definition).